packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -638,8 +661,35 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
       e: React.SyntheticEvent,
       searchInputProps: DropdownSearchInputProps,
     ) => {
-      if (keyboardKey.getCode(e) === keyboardKey.Tab && !_.isNil(highlightedIndex)) {
-        selectItemAtIndex(highlightedIndex)
+      switch (keyboardKey.getCode(e)) {
+        case keyboardKey.Tab:
+          if (!_.isNil(highlightedIndex)) {
+            selectItemAtIndex(highlightedIndex)
+          }
+          break
+        case keyboardKey.ArrowLeft:
```

Comment:
I don't see a handler for the right key, is that intentional? Also, in RTL, right and left keys need to be swapped.